client/src/screens/schedules/Detail.js

```diff
@@ -46,6 +46,7 @@ class Detail extends Component {
     modalVisible: false,
     modalStartTime: 0,
     modalEndTime: 0,
+    refreshing: false,
```

Comment:
can you use the loading prop from the apollo query?

Reply:
I can try

Reply:
Loading prop is ready only (all props are)